src/Google.Maps/TravelMode.cs

```diff
@@ -24,6 +24,11 @@ public enum TravelMode
 		/// <summary>
 		/// requests bicycling directions via bicycle paths & preferred streets (currently only available in the US).
 		/// </summary>
-		bicycling
-	}
+		bicycling,
+
+        /// <summary>

```

Comment:
Hey, looks like the whitespace is a little off here? I believe these files use tabs instead of spaces.